cfgov/unprocessed/css/atoms/partially-styled-link.less

```diff
@@ -0,0 +1,50 @@
+/* topdoc
+  name: Partially styled link
+  family: cfgov-atoms
+  patterns:
+    - name: Default example
+      markup: |
+          <a class="a-link a-link__partially-styled">
+            <span class="a-link__partially-styled_plain">
+              Unstyled text
+            </span>
+            <span class="a-link__partially-styled_underlined">
+              Underlined text
+            </span>
+          </a>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .a-link.a-link__partially-styled
+            span.a-link__partially-styled_plain
+            span.a-link__partially-styled_underlined
+  tags:
+    - cfgov-atoms
+*/
+
+.a-link__partially-styled {
+    border-bottom-width: 0px;
+  
+    &_plain {
+        color: @black;
+        border-bottom: 1px dotted transparent;
+    }
+  
+    &_underlined {
+        border-bottom: 1px dotted @link-underline;
+    }
+  
+    &:visited &_underlined {
+        border-bottom: 1px dotted @link-underline-visited;
+    }
+  
+    &:hover span,
+    &:focus span {
+        border-bottom: 1px solid @link-underline-hover !important;
```

Comment:
It looks like to override the visited border color on the underlined part of the link when it's hovered:

<img width="330" alt="screen shot 2017-05-03 at 8 58 47 am" src="https://cloud.githubusercontent.com/assets/778171/25669581/cae720c2-2fde-11e7-860d-386191b27892.png">

vs

<img width="332" alt="screen shot 2017-05-03 at 8 46 29 am" src="https://cloud.githubusercontent.com/assets/778171/25669528/a05dfe66-2fde-11e7-87a5-344aa5022e3a.png">